std_msgs/MultiArrayLayout layout

float64[] data

int32[] index2value

string who_detect

int32[] valid_img